docs/content/users/usage/cms-settings.md

```diff
@@ -70,6 +70,14 @@ As per [Drush docs](https://www.drush.org/13.x/commands/#xdebug) you can:
 * Enable it for a single Drush command by running: `ddev drush --xdebug`
 * Set `DRUSH_ALLOW_XDEBUG=1` [environment variable](../extend/customization-extendibility.md#environment-variables-for-containers-and-services), allowing every Drush call to be run with Xdebug when the PHP extension is enabled.
 
+#### Composer and Xdebug
```

Comment:
This section `#### Composer and Xdebug` is added under `### Drupal Specifics` - but there is no relation to Drupal.

Reply:
I regroup details in "step debugging" and add a info link into Built-in Developer Tools > DDEV and Composer